### DISCUSSION TITLE
Remove mandatory options

### DISCUSSION THREAD

**Discussion by rajun918** [cvat-ai/cvat#4222] [Ideas]
Hi team,
How to remove mandatory options of mentioning number points required while marking polyline.

**Comment by nmanovic** [comment]
@rajun918 , the number of points for a polygon is an optional parameter. You can keep it empty.

**Reply by rajun918** [reply]
Thanks for your replay @nmanovic If keep it empty we can draw polygon but not ended polygon.

**Comment by nmanovic** [comment]
@rajun918 , you need to press 'n' one more time. Please read documentation. All these features should be described pretty well.
https://openvinotoolkit.github.io/cvat/docs/manual/advanced/annotation-with-polygons/

**Reply by rajun918** [reply]
Thanks @nmanovic
